examples/imitation_learning_openbot/generate_episodes.py

```diff
@@ -16,22 +16,23 @@
 
     parser = argparse.ArgumentParser()
     parser.add_argument("--num_episodes_per_scene", type=int, default=10)
+    parser.add_argument("--num_candidate_points_per_episode", type=int, default=10)
     parser.add_argument("--episodes_file", default=os.path.realpath(os.path.join(os.path.dirname(__file__), "train_episodes.csv")))
     parser.add_argument("--scene_id")
     args = parser.parse_args()
 
-    # directory for trajectory image storage
-    split_name = os.path.splitext(os.path.basename(args.episodes_file))[0] # isolate filename and remove extension
+    # create directory for storing episodes information
+    file_name = os.path.splitext(os.path.basename(args.episodes_file))[0] # isolate filename and remove extension
     episodes_dir = os.path.realpath(os.path.join(os.path.dirname(__file__), "episodes"))
-    split_dir = os.path.realpath(os.path.join(episodes_dir, split_name))
-    os.makedirs(split_dir, exist_ok=True)
+    output_dir = os.path.realpath(os.path.join(episodes_dir, file_name))
```

Comment:
- for simplicity, just write the the plots directly to `episodes_dir`

Reply:
Updated this